### PR TITLE
APPS:6120 updated module dependencies with new versions

### PR DESCRIPTION
**Pull request by oleksander-kiiashko**
Task: https://spryker.atlassian.net/browse/APPS-6120

### CODE REVIEW

File: docs/pbc/all/payment-service-providers/payone/payone.md [hunk 1]
```diff
@@ -79,10 +79,13 @@ When customers pay with PayPal, a shop owner can do the following:
...
+* `spryker/payment: ^5.10.0`
 * `spryker/sales: ^11.32.0`
 * `spryker/sales-return: ^1.4.0`
-* `spryker-shop/checkout-page: ^3.20.0`
-* `spryker-shop/payment-page: ^1.0.0`
+* `spryker-shop/checkout-page: ^3.21.0`
+* `spryker-shop/payment-page: ^1.2.0`
+* `spryker/oms: ^11.21.0`
+* `spryker/sales-payment: ^1.2.0`
+* `spryker/sales-oms: ^0.1.1`
```
Comment (oleksander-kiiashko) [inline docs/pbc/all/payment-service-providers/payone/payone.md:91]: it was added with Order State Machine improvements
https://github.com/spryker/spryker/pull/9462

Comment (matweew) [reply docs/pbc/all/payment-service-providers/payone/payone.md:91]: What functionality from this module is used in PayOne app?

File: docs/pbc/all/ratings-reviews/202204.0/third-party-integrations/integrate-bazaarvoice.md [hunk 2]
```diff
@@ -8,15 +8,21 @@ template: howto-guide-template
...
 * `spryker-shop/asset-widget: ^1.0.0`
+* `spryker-shop/cart-page: ^3.32.0`
 * `spryker-shop/product-detail-page: ^3.17.0`
 * `spryker-shop/product-category-widget: ^1.6.0`
-* `spryker-shop/shop-ui: ^1.59.0`
+* `spryker-shop/shop-ui: ^1.62.0`
+* `spryker-shop/checkout-page: ^3.23.0`
+* `spryker-shop/merchant-page: ^1.1.0`
+* `spryker-shop/merchant-profile-widget: ^1.1.0`
+* `spryker-shop/merchant-widget: ^1.3.0`
```
Comment (matweew) [inline docs/pbc/all/ratings-reviews/202204.0/third-party-integrations/integrate-bazaarvoice.md:24]:
```suggestion
* `spryker-shop/merchant-page: ^1.1.0`  (Marketplace only)
* `spryker-shop/merchant-profile-widget: ^1.1.0` (Marketplace only)
* `spryker-shop/merchant-widget: ^1.3.0` (Marketplace only)
```

Comment (danielsantos-spryker) [reply docs/pbc/all/ratings-reviews/202204.0/third-party-integrations/integrate-bazaarvoice.md:24]: nice catch, besides this LGTM.

File: docs/pbc/all/payment-service-providers/payone/payone.md (hunk 1 above)
Comment (oleksander-kiiashko) [reply docs/pbc/all/payment-service-providers/payone/payone.md]: fixed, it has no relation to PayOne app

**Review by matweew — APPROVED** [review]

**Review by danielsantos-spryker — DISMISSED** [review]

**Review by lenadoc — APPROVED** [review]

**Pull request merged**